examples/Pie.ipynb

```diff
@@ -28,20 +28,21 @@
    "outputs": [],
```

Comment:
Lets add a markdown cell above that says "Here we generate the simplest possible pie chart by passing the data as the sizes we want each slice to be"?